net-libs/gnome-online-accounts/metadata.xml

```diff
@@ -7,6 +7,7 @@
   </maintainer>
   <use>
     <flag name="gnome">Use <pkg>gnome-base/gnome-control-center</pkg> to set GNOME online accounts preferences</flag>
+    <flag name="ms-graph">Enable Microsoft 365 provider</flag>
```

Comment:
I would name the USE flag "ms365". I think most end-users will understand better what that USE flag is for than understanding " ms-graph" meaning

Reply:
Done in https://github.com/gentoo/gentoo/commit/20171033b2493ec07f0e848030c3200b6e3d66fb